compiler/luci/pass/src/PropagateQParamBackwardPass.cpp

```diff
@@ -188,8 +190,6 @@ void propagate_pack_quantparam(luci::CirclePack *pack)
       if (succs.size() > 1)
         continue;
 
-      // Non-const input must have been quantized
-      assert(node->quantparam() != nullptr);
```

Comment:
This pass is now used for `QuantizeOnnxFakeQuantModelPass`, which makes this assumption invalid.

Q) Is it safe to remove the assert?
A) I think yes. The assert was just for checking the behavior of `QuantizeWithMinMaxPass`. Even though `node->quantparam() == nullptr`, `overwrite_quantparam` can work without any problem.